cypress/plugins/index.js

```diff
@@ -0,0 +1,21 @@
+/// <reference types="cypress" />
```

Comment:
Is this file needed?

Reply:
This one was also Cypress-generated, so would prefer to keep for future use.